src/main/java/com/leahtcodes/Controllers/UserController.java

```diff
@@ -19,6 +28,27 @@ public UserController(UserRepository userRepository){
         this.userRepository = userRepository;
     }
 
+    @Autowired
```

Comment:
Really want to stay from doing this. Sometimes, it might be required, but ONLY for depreciated code.  I would update this to use your constructor again. 

Move this back up to the top and bring this into your constructor. Like you did the CoffeeController